fendermint/app/src/cmd/rpc.rs

```diff
@@ -40,6 +42,8 @@ use super::key::read_secret_key;
 
 cmd! {
   RpcArgs(self) {
+    let _trace_file_guard = set_global_tracing_subscriber(&TracingSettings::default());
```

Comment:
Do we need this inside every command? Why not in the main entrypoint in `fendermint/app/src/main.rs`?

Reply:
Because at that point we don't have the config file loaded. But I agree this is a bit annoying! The other place where I can move this would be here in the match: https://github.com/consensus-shipyard/ipc/blob/05f5149aeb611f6606a35f516f4caa5d08b8ca57/fendermint/app/src/cmd/mod.rs#L64

I just though it might be cleaner to have in each command but no strong feelings about it. So happy to move it. WDYT?

Reply:
I prefer to factor boilerplate out to a common place. It's way too easy to implement a new command and forget about this.